serveradmin/api/api.py

```diff
@@ -1,58 +0,0 @@
-import random
```

Comment:
Seems like it would be just some debug functions? Would also be interesting to know why they were there and why you remove them now. 😉

Reply:
Me too.  I guess we will deploy and learn.